activerecord/test/cases/tasks/database_tasks_test.rb

```diff
@@ -812,6 +814,34 @@ def capture_migration_output
           end
         end
     end
+
+    class DatabaseTasksMigrateStatusTest < DatabaseTasksMigrationTestCase
+      test "migrate status table" do
```

Comment:
I know this varies from file to file, but I think it is better to maintain the same style as in the other tests, at least in the same file, I would therefore declare this as:
```ruby
def test_migrate_status_table
```

The same applies to:
```ruby
def migrate_status_table_without_migrations_table
```

Reply:
My mistake, I noticed they were using `setup do..end` and `teardown do..end`, so I assumed they were also using `test "desc" do..end` as well. Thanks for pointing that out 👍